indico/modules/events/contributions/models/fields.py

```diff
@@ -115,6 +115,10 @@ def field(self):
     def mgmt_field(self):
         return self._get_field(management=True)
 
+    @property
+    def filter_choices(self):
+        return {x['id']: x['option'] for x in self.field_data['options']} if self.field_data.get('options') else {}
```

Comment:
In case `self.field_data['options']` is always a dict (i.e. not `None` etc) you can simplify this:

    return {x['id']: x['option'] for x in self.field_data.get('options', {})}